test/system/700-play.bats

```diff
@@ -988,52 +988,94 @@ _EOF
 }
 
 @test "podman play with automount volume" {
-    imgname="automount-test-$(safename)"
-    podname="p-$(safename)"
-    ctrname="c-$(safename)"
-
-    cat >$PODMAN_TMPDIR/Containerfile <<EOF
+    imgname1="automount-test-$(safename)1"
+    imgname2="automount-test-$(safename)2"
```

Comment:
Suggestion: `automount-test1-$(safename)` and `-test2-`. The way they are now, 1 and 2 appear at the end of a random string, making it impossible for a human to see the difference between them.